qiskit_nature/second_q/algorithms/initial_points/hf_initial_point.py

```diff
@@ -37,46 +34,23 @@ class HFInitialPoint(InitialPoint):
     :class:`~qiskit.circuit.library.initial_states.hartree_fock.HartreeFock` initial state (which
     will be prepended to the :class:`~qiskit.circuit.library.ansatzes.ucc.UCC` circuit) the all-zero
     initial point will correspond to the HF initial point.
-
-    The excitation list generated by the :class:`~qiskit.circuit.library.ansatzes.ucc.UCC` ansatz is
-    obtained to ensure that the shape of the initial point is appropriate.
     """
 
     def __init__(self) -> None:
         super().__init__()
         self._ansatz: UCC | None = None
         self._excitation_list: list[tuple[tuple[int, ...], tuple[int, ...]]] | None = None
+        self._reps: int = 1
```

Comment:
Is there any reason to pass these in and store them. Once the anstaz is known they can be gotten from the ansatz right - thats what done internally. Why copy them over when you can access them of ansatz?

Reply:
I thought the same, but I was already pulling the excitation list out, so I chose to take the reps too. But, as you pointed it out, I'll just access these via the ansatz :).

Reply:
Is the only reason you were pulling out the excitation list, its usage in the MP2 ansatz, in the first place? If so, maybe we can remove its duplication there too, and just access it via the ansatz everywhere :thinking: